ConfigurationSystem/Client/Utilities.py

```diff
@@ -544,3 +544,42 @@ def getDBParameters( fullname ):
   parameters[ 'DBName' ] = dbName
 
   return S_OK( parameters )
+
+def getElasticDBParameters( fullname ):
+  """
+  Retrieve Database parameters from CS
+  fullname should be of the form <System>/<DBname>
+  
+  """
+
+  cs_path = getDatabaseSection( fullname )
+  parameters = {}
+
+  result = gConfig.getOption( cs_path + '/Host' )
+  if not result['OK']:
+    # No host name found, try at the common place
+    result = gConfig.getOption( '/Systems/Databases/Host' )
```

Comment:
This is the default place holder for SQL databases, shouldn't we have another default for elastic search ?